.ci/docker/iis/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM mcr.microsoft.com/dotnet/framework/aspnet
```

Comment:
this Docker image is not needed for tests, but it is an environment where the test can run, we invest time to make it and I think it worth to be keep

Reply:
Sounds good, what about a README.md to explain this particular Dockerfile is not used but it could be, and if so, what commands are required to use it locally?